Source/trigs.cpp

```diff
@@ -89,25 +87,25 @@ int L4PentaList[33] = {
 
 void InitNoTriggers()
 {
-	trigflag_4 = 0;
-	trigflag_3 = 0;
+	numtrigs = 0;
+	trigflag = 0;
 }
 
 void InitTownTriggers()
 {
-	char v0; // bl
-	int v1;  // eax
-	int v2;  // eax
+	int i;
 
 	trigs[0]._tx = 25;
 	trigs[0]._ty = 29;
 	trigs[0]._tmsg = WM_DIABNEXTLVL;
-	trigflag_4 = 1;
-	if (gbMaxPlayers == 4) {
+
+	numtrigs = 1;
+
+	if(gbMaxPlayers == 4) {
+		for(i = 0; i < 3; i++) {

```

Comment:
Maybe a good idea to get the size of the array instead of hardcoded amount of loops